### PR TITLE
 Address NB issue #1283 and add named scenarios for NB Pulsar and NB Kafka adapters

### PR DESCRIPTION
**Pull request by yabinmeng**
Issue #1283 detail

### CODE REVIEW

**Review by jshook — APPROVED** [review]
I didn't see anything amiss. Thanks for updating the workloads!

File: adapter-pulsar/src/main/resources/pulsar.md [hunk 1]
```diff
@@ -1,8 +1,25 @@
-# pulsar
+---
+weight: 0
+title: S4R
```
Comment (dave2wave) [inline adapter-pulsar/src/main/resources/pulsar.md:3]: Should the title be Pulsar?

File: adapter-kafka/src/main/resources/s4k.md [hunk 2]
```diff
@@ -1,22 +1,33 @@
-# Overview
+---
+weight: 0
+title: S4J
```
Comment (dave2wave) [inline adapter-kafka/src/main/resources/s4k.md:3]: Should the title be S4K?

Comment (yabinmeng) [reply adapter-kafka/src/main/resources/s4k.md:3]: fixed. thanks

File: adapter-pulsar/src/main/resources/pulsar.md (hunk 1 above)
Comment (yabinmeng) [reply adapter-pulsar/src/main/resources/pulsar.md:3]: fixed

**Pull request merged**